Sources/MessagingInApp/Views/InAppMessageView.swift

```diff
@@ -150,26 +150,9 @@ public class InAppMessageView: UIView {
         }
 
         // If a different message is currently being shown, we want to replace the currently shown message with new message.
-        if let currentlyDisplayedInAppWebView = inlineMessageManager?.inlineMessageView, messageRenderingLoadingView == nil {
-            // To provide the user with feedback indicating a new message is being rendered, show an activity indicator while the new message is loading.
-            let activityIndicator = UIActivityIndicatorView(style: .large)
-            activityIndicator.startAnimating()
-            activityIndicator.isHidden = true // start hidden so when we add the subview, it does not cause a flicker in the UI. Wait to show it when the animation begins.
-
-            addSubview(activityIndicator)
-            assert(messageRenderingLoadingView != nil, "Expect activity indicator to be added as a subview")
-
-            // Set autolayout constraints to position the activity indicator.
-            activityIndicator.translatesAutoresizingMaskIntoConstraints = false
-            NSLayoutConstraint.activate([
-                activityIndicator.centerXAnchor.constraint(equalTo: centerXAnchor),
-                activityIndicator.centerYAnchor.constraint(equalTo: centerYAnchor),
-                activityIndicator.widthAnchor.constraint(equalTo: widthAnchor),
-                activityIndicator.heightAnchor.constraint(equalTo: heightAnchor)
-            ])
-
-            animateFadeInOutInlineView(fromView: currentlyDisplayedInAppWebView, toView: activityIndicator) {
```

Comment:
All of this code is to animate an activity indicator when a close button is pressed. We can re-use all of this code for this feature when a "show another message" action button is clicked. So I refactored the View to make this code re-usable.